### DISCUSSION TITLE
Just curious re: releases not being in .gitignore

### DISCUSSION THREAD

**Discussion by vidjuheffex** [biwascheme/biwascheme#263] [Q&A] [answered]
with the ability to download them from the repo's release page, install biwa from npm, run `npm run build` etc, seems that it'd be better to not have them in the version control.

(I'm not the best at setting up my dev enviroment but this mostly stems from frustration at the releases folder showing up in my "find in files" so perhaps there is a fix on my end for this friction)

**Comment by yhara** [comment, marked as answer]
At least the latest file (right now it's release/biwascheme-0.7.4.js) needs to be committed because it is linked from the website (which is hosted by github pages.)

However, older versions can be safely removed from the repo.
